printer/kubectl_describe.go

```diff
@@ -20,14 +20,19 @@ type DescribePrinter struct {
 	tableBytes *bytes.Buffer
 }
 
+var onlyValuePathToColor = regexp.MustCompile(`^(Labels|Annotations|(Init )?Containers/[^/]+/Environment Variables from)/.+`)
+
 func (dp *DescribePrinter) Print(r io.Reader, w io.Writer) {
 	scanner := describe.NewScanner(r)
 	const basicIndentWidth = 2 // according to kubectl describe format
 	for scanner.Scan() {
 		line := scanner.Line()
 
-		// when there are multiple columns, treat is as table format
-		if bytesutil.CountColumns(line.Value, " \t") >= 3 {
+		if onlyValuePathToColor.MatchString(scanner.Path().String()) { // if line path matches label or annotation or env from
+			line.Value = bytes.Join([][]byte{line.Key, line.Value}, line.Spacing)
+			line.Key = nil
+			line.Spacing = nil
```

Comment:
I just had a thought. Won't this break on the first line of Labels or Annotations?

I just tried out your branch, and it seems to work just fine. Why isn't it breaking?

Maybe we want to edit the scanner/describe/describe.go to fix this invalid key-value handling instead :thinking:

Reply:
Because the regex will not match the first line of label or annotation.